README.md

```diff
@@ -23,7 +25,9 @@ Create a config.json file in this directory. Below is a good default config.
 ```js
 {
     "emailRecipients": ["email.receiver1@gmail.com", "email.receiver2@gmail.com"],
-    "slackChannelName": "node-monitor",
+    "slackChannelName": "your-node-monitor-channel",
```

Comment:
Let's take this out since we're not integrating slack anymore

Reply:
It was WhatsApp that we were not going to use.

Channels that we will use:
- Email
- Slack
- Telegram